modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding-upgrade-viability-conditiontype_{context}"]
+= Understanding upgrade viability with conditionType
+
+The status of cluster operators is revealed with the value of conditionType, informing you of potential impacts to upgrading your cluster. 
+This information can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
+
+[source,terminal]
+----
+$ oc get clusterversion -o YAML
```

Comment:
Most people will not capitalize the YAML in the command (it seems that the rest of the docs use `-o yaml` in these listings, too)